js/script.js

```diff
@@ -582,11 +582,11 @@ function ot_buildMatcher() {
 
 var ot_validCombos = {
 
-    "ot_linux,ot_large_model,ot_python,ot_X64,ot_CUDA,ot_stable":
-    "pip install onnxruntime-training -f https://download.onnxruntime.ai/onnxruntime_stable_<b>&lt;cu_version*</b>&gt;.html<br/>pip install torch-ort<br/>python -m torch_ort.configure<br/><br/>*</b><a href='https://download.onnxruntime.ai/' target='blank'>Available versions</a>",
+    "ot_linux,ot_large_model,ot_python,ot_X64,ot_CUDA118,ot_stable":
+    "pip install --pre onnxruntime-training --extra-index-url=https://aiinfra.pkgs.visualstudio.com/PublicPackages/_packaging/ORT/pypi/simple/<br/>pip install torch-ort<br/>python -m torch_ort.configure",
```

Comment:
@snnn is there any guideline around using `--extra-index-url` as installation instructions to give to our users?